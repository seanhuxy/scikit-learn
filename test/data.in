1 2.0 1
3 8.1 0
5 1.4 1
5 2 0
4 29 0
