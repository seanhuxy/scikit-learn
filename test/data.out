1.0 2.0 
3.0 8.1 
0.0 -1.4 
0.0 2.0 
99.0 -29.0 
nan nan 
